From 5bcf691759e4ca2860db1711b1010fa772aacb67 Mon Sep 17 00:00:00 2001
From: Evilebot Tnawi <evilebottnawi@users.noreply.github.com>
Date: Wed, 5 Dec 2018 16:22:35 +0300
Subject: [PATCH 07/19] fix: handle uppercase `IMAGE-SET`

Just find problem place in code when look implementation, sorry don't have time on tests
---
 index.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/index.js b/index.js
index 3775a02..bae3144 100644
--- a/index.js
+++ b/index.js
@@ -2,7 +2,7 @@ import postcss from 'postcss';
 import valueParser from 'postcss-values-parser';
 import processImageSet from './lib/process-image-set';
 
-const imageSetValueMatchRegExp = /(^|[^\w-])(-webkit-)?image-set\(/
+const imageSetValueMatchRegExp = /(^|[^\w-])(-webkit-)?image-set\(/i
 const imageSetFunctionMatchRegExp = /^(-webkit-)?image-set$/i
 
 export default postcss.plugin('postcss-image-set-function', opts => {
-- 
2.33.1

